src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -77,6 +80,27 @@ public StorageFile(String filePath) throws InvalidStorageFilePathException {
     private static boolean isValidPath(Path filePath) {
         return filePath.toString().endsWith(".txt");
     }
+    
+    
+    /**
+     * Return true if the file is deleted.
+     */
+    public boolean isFileDeleted() {
+    	File f = new File(path);
+    	if(f.exists()) {
+    		return false;
+    	}
+    	else {
+    		return true;
+    	}
```

Comment:
- This if-else block is not necessary, and can be replaced with a single return statement.
- The if-else block's formatting does not follow the coding standard.